src/expect4clj.clj

```diff
@@ -0,0 +1,20 @@
+(ns expect4clj
```

Comment:
my first attempt was to use Expect4j, but decided to do without it. Will get rid of this.